src/libfetchers/git.cc

```diff
@@ -391,10 +397,20 @@ struct GitInputScheme : InputScheme
                 utimes(localRefFile.c_str(), times);
             }
 
+            // don't know how to check if the lfs was cloned up to the end, so fetch it all the time (if enabled, which should be uncommon)
+            if (largeFileSystem) {
+                if (getProgramStatus("git", true, { "-C", repoDir, "remote", "get-url", "nix_fetcher_remote_for_lfs"})) {
```

Comment:
Is there a better way to do that ? In particular, the user see an error logged on stderr if the remote doesn't exist (but I put it before it run the git lfs fetch command to prevent issue if interupted, as git fail to create a remote if one already exist).